app.py

```diff
@@ -0,0 +1,30 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
```

Comment:
Secret key should be in .env file and .gitignore should be added to make sure it is hidden